api/build/create.go

```diff
@@ -51,24 +49,32 @@ import (
 //   - ApiKeyAuth: []
 // responses:
 //   '200':
-//     description: Request processed but build was skipped
+//     description: Successfully received the webhook but build was skipped
```

Comment:
you can create builds without webhooks, so i don't know if it makes sense to single out the webhook scenario here (and elsewhere).